packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -647,7 +647,7 @@ class AnimationController extends Animation<double>
   void stop({ bool canceled = true }) {
     _simulation = null;
     _lastElapsedDuration = null;
-    _ticker.stop(canceled: canceled);
+    _ticker?.stop(canceled: canceled);
```

Comment:
Why don't you check null for all statement which use `_ticker`?

Reply:
As per my understanding in the other places of the controller where `_ticker` is used is the ticker is actually necessary, so a null check would actually behave like things are working when they're not.

In the case of the `stop()` if the `_ticker` was already disposed the `stop()` has fulfilled his purpose.